ext/shmop/shmop.c

```diff
@@ -66,22 +59,59 @@ zend_module_entry shmop_module_entry = {
 ZEND_GET_MODULE(shmop)
 #endif
 
-/* {{{ rsclean
- */
-static void rsclean(zend_resource *rsrc)
+zend_class_entry *shmop_ce;
+static zend_object_handlers shmop_object_handlers;
+
+static inline php_shmop *shmop_from_obj(zend_object *obj)
+{
+	return (php_shmop *)((char *)(obj) - XtOffsetOf(php_shmop, std));
+}
```

Comment:
if we are exposing php_shmop(not sure if there will be real usecase to access this though), than it’s better to also expose this.

Reply:
Good catch; I didn't think of this before, but probably it's better to not expose `php_shmop` at all.

Reply:
it’s also okey. :)

Reply:
Oops, almost forgot about that. Now done.